gettsim/benefits/elterngeld.py

```diff
@@ -0,0 +1,36 @@
+def elt_geld(person, params):
+    """This function calculates the monthly benefits for having
+    a child that is up to one year old (Elterngeld)"""
+
+    if person["elt_zeit"]:
+        considered_wage = person["m_wage_l1"] - person["m_wage"]
+
```

Comment:
you definitely need net wage. you can just feed this into the tax tariff, similar to how it is done with the unemployment benefit. social sec. contributions also need to subtracted.

Reply:
Yeah about that. I was really wondering if we want to do this every time for some functions. I mean we have a detailed ssc function and then just use and approximation for alg and then maybe also here...Shouldn't we rely on our code instead of approximations?

Reply:
We could use the values from the alg call of course. It's the same lagged wage.

Reply:
I renamed and reused the function.